services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/BeanTableSchema.java

```diff
@@ -203,6 +206,19 @@ private BeanTableSchema(StaticTableSchema<T> staticTableSchema) {
         return builder.build();
     }
 
+    private static AttributeConfiguration resolveAttributeConfiguration(PropertyDescriptor propertyDescriptor) {
```

Comment:
I like this encapsulation of the configuration settings.

Reply:
+1